Software/Development/SemesterProject/App/Include/encoderManager.h

```diff
@@ -13,13 +13,13 @@
 
 #define ENCODER_STEPS 2048
 #define ENCODER_STEPS_INVERSE ((float)(1.0 / ENCODER_STEPS))
-#define BASE_FREQ 50        //us
+#define POLE_PAIRS  2   //Number of pole pairs of the motor.
+#define POLE_PAIRS_INVERSE ((float) 0.5)    //To be used instead of a division.
```

Comment:
Only thing you could consider is to move the defines to constants.h

Reply:
done